Procfile

```diff
@@ -0,0 +1 @@
+web: ./carnival
```

Comment:
You added one to periodically scrub the example comments on the homepage, I think.

https://github.com/thoughtbot/carnival/blob/master/carnival.cabal#L153